web/src/reducers/apd.js

```diff
@@ -45,6 +56,7 @@ const reducer = (state = initialState, action) => {
         fetching: false,
         loaded: true,
         data: {
+          ...state.data,
```

Comment:
Not sure about this.  It creates a scenario where the local data could supersede the server data.  I don't think it's something that should realistically happen, but...  🤷‍♂️?  The reason for the spread in the root object is that the `fetching`, `loading`, etc. properties are client-specific metadata and not represented in the server data at all.

What's your thinking here?

Reply:
yeah, it would supercede if the server data didn't include a particular key/value (the idea being so that we're guaranteed to have at least the initial keys in the apd/activity object), but i agree, it's still weird and not sure what i was thinking -- i updated :)